templates/corporate/attribution.html

```diff
@@ -30,16 +30,6 @@ <h1>Website attributions</h1>
                         <img alt="" src="{{ static('images/landing-page/zulip-cloud/support.svg') }}" />
                         <p><a href="https://storyset.com/work">Work illustrations by Storyset</a></p>
                     </li>
-                    <li>
-                        <b>On <a href="/role/engineers/">/role/engineers/</a> page:</b>
-                        <img alt="" src="{{ static('images/landing-page/zulip-cloud/support.svg') }}" />
-                        <p><a href="https://storyset.com/work">Work illustrations by Storyset</a></p>
-                    </li>
-                    <li>
-                        <b>On <a href="/role/engineers/">/role/engineers/</a> page:</b>
-                        <img alt="" src="{{ static('images/landing-page/zulip-cloud/support.svg') }}" />
-                        <p><a href="https://storyset.com/work">Work illustrations by Storyset</a></p>
```

Comment:
I think it's nice to attribute even though it's not legally required by their terms. Among other things, it makes it easy for us to verify that we have the rights to use the image. So I'd prefer to just correct the URLs to correctly reference the actual images we got from there.